client/src/components/Feed/User.js

```diff
@@ -0,0 +1,395 @@
+// Core
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { Card } from "antd-mobile";
+import styled from "styled-components";
+import { escapeRegExp } from "lodash";
+import { useTranslation } from "react-i18next";
+
+// Local
+import Heading from "components/Typography/Heading";
+import PostCard from "./PostCard";
+import FilterTag from "components/Tag/FilterTag";
+import TextAvatar from "components/TextAvatar";
+import { buildLocationString } from "./utils";
+import { getInitialsFromFullName } from "utils/userInfo";
+import {
+  PlaceholderIcon,
+  IconsContainer,
+  SocialIcon,
+} from "../Profile/ProfileComponents";
+import { mq } from "constants/theme";
+import {
+  FACEBOOK_URL,
+  INSTAGRAM_URL,
+  LINKEDIN_URL,
+  LINKEDIN_INDIVIDUAL_URL,
+  TWITTER_URL,
+  GITHUB_URL,
+  APPSTORE_URL,
+  PLAYSTORE_URL,
+} from "constants/urls";
+import { LOGIN } from "templates/RouteWithSubRoutes";
+
+// Icons
+import SvgIcon from "../Icon/SvgIcon";
+import statusIndicator from "assets/icons/status-indicator.svg";
+import envelopeBlue from "assets/icons/mail.svg";
+import facebookIcon from "assets/icons/social-facebook-unfilled.svg";
+import instagramIcon from "assets/icons/social-instagram-unfilled.svg";
+import githubIcon from "assets/icons/social-github.svg";
+import linkedinBlue from "assets/icons/social-linkedin-unfilled.svg";
+import twitterBlue from "assets/icons/social-twitter-unfilled.svg";
+import websiteIcon from "assets/icons/social-website-blue.svg";
+import playSotreIcon from "assets/icons/google-play.svg";
```

Comment:
Not sure who the designer is but we should use only one icon file and be consistent.

* Google Play Store: client/src/assets/icons/play-store-icon.svg is used in organisation profile page - these seem the same or close we should use the existing one for both
* Apple App Store: client/src/assets/icons/app-store-icon.svg is used in organisation profile page, but I think the one added here is the more "legit" one. So maybe we should use that for both?

Also there is a typo in both - "S**ot**re"